src/containers/Home.jsx

```diff
@@ -1,11 +1,14 @@
 import React, { Component } from 'react';
 import { Container } from 'react-bootstrap';
 
+import VideoFeed from './VideoFeed';
```

Comment:
If we're using `VideoFeed` inside `Home`, then `VideoFeed` should be in `src/components/`